sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -44,37 +45,23 @@ class UnknownIdentifier(object):
     def __init__(self, identifier):
         self.identifier = identifier
 
-class MicrosoftTeamsUserIdentifier(object):
-    """
-    Represents an identifier for a Microsoft Teams user.
-    :ivar user_id: the string identifier representing the identity
-    :vartype user_id: str
-    :param user_id: Value to initialize MicrosoftTeamsUserIdentifier.
-    :type user_id: str
-    :ivar is_anonymous: set this to true if the user is anonymous for example when joining a meeting with a share link
-    :vartype is_anonymous: bool
-    :param is_anonymous: Value to initialize MicrosoftTeamsUserIdentifier.
-    :type is_anonymous: bool
-    """
-    def __init__(self, user_id, is_anonymous=False):
-        self.user_id = user_id
-        self.is_anonymous = is_anonymous
-
 class CommunicationIdentifierModel(msrest.serialization.Model):
     """Communication Identifier Model.
 
     All required parameters must be populated in order to send to Azure.
 
     :param kind: Required. Kind of Communication Identifier.
     :type kind: CommunicationIdentifierKind
-    :param id: identifies the Communication Identitity.
+    :param id: Full id of the identitifier.
     :type id: str
     :param phone_number: phone number in case the identity is phone number.
```

Comment:
Same comment around consistency in casing after :

```suggestion
    :param phone_number: The phone number in case the identity is a phone number.
```